src/nestdoorapp/urls.py

```diff
@@ -5,13 +5,12 @@
 from django.conf.urls.static import static
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
```

Comment:
Pls review this line

Reply:
I've added it back in my source code which has been pushed to the branch